pkg/controllers/management/node/utils.go

```diff
@@ -227,7 +227,7 @@ func (m *Lifecycle) reportStatus(stdoutReader io.Reader, stderrReader io.Reader,
 			// under trace and don't add to the v3.NodeConditionProvisioned.Message
 			logrus.Tracef("[node-controller] %v", msg)
```

Comment:
We probably also want to include the new prefix here, for consistency

Reply:
Yeah that makes sense. Updated.